src/assets/toolkit/styles/components/sky.css

```diff
@@ -238,15 +268,27 @@
  * 1. At large enough screens, make the menu _always_ visible.
  * 2. Remove the bump (since it won't be above anything at this point).
  * 3. Reset the background.
- * 4. Reset the order.
+ * 4. To align right, this now needs to be last.
+ * 5. Constraining the width keeps the last three links on a second line to avoid orphans.
  */
 
 @media (--Sky-nav-expanded-viewport) {
   .Sky-nav-menu {
     display: block; /* 1 */
     padding-bottom: 0; /* 2 */
     background: transparent; /* 3 */
-    order: 0; /* 4 */
+    order: 2; /* 4 */
```

Comment:
I'm not sure I'm with you yet, @tylersticka  
on narrow screens, the order is:
-1: nav
0: logo (this is just by default, it isn't actually defined)
1: sky-nav-controls

On expanded screens, the order is:
0: logo
1: sky-nav-controls
2: nav

I'm not sure what resetting the order of sky-nav-controls on expanded screens would get me?